cli_e2e/plan_storage_test.go

```diff
@@ -50,3 +51,40 @@ func TestGCPPlanStorageStorageAndRetrieval(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, readContents, contents)
 }
+
+func TestAWSPlanStorageStorageAndRetrieval(t *testing.T) {
+	fmt.Printf("in function")
```

Comment:
nit: maybe remove this print, and while here also the one int he func above hehe